CONTRIBUTING.md

```diff
@@ -53,137 +53,23 @@
 - All functions, methods, classes and attributes should be documented with a docstring. We use the
   [`Google docstring style`](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). If you come
   across a function or method that doesn't conform to this standard, please update it as you go
-- When adding a new public interface, it has to be  included in the reference documentation located in
-  `docs/reference`. If applicable, add or modify examples in the docs related to the new functionality implemented,
-  following the guidelines established in [Adding Examples](#adding-examples)
 
 ### Writing and running tests
 
-Tests are contained within the `tests` directory, and follow the same directory structure as the `starlite` module.
+Tests should be contained within the `tests` directory, and follow the same directory structure as the `starlite` module.
 If you are adding a test case, it should be located within the correct submodule of `tests`. E.g. tests for
-`starlite/utils./sync.py` reside in `tests/utils/test_sync.py`.
+`starlite/utils/sync.py` reside in `tests/utils/test_sync.py`.
 
-The `Makefile` includes several commands for running tests:
+If the repository you are working on includes a `Makefile` you have access to includes several commands for running tests:
 
 - `make test` to run tests located in `tests`
 - `make test-examples` to run tests located in `docs/examples/tests`
 - `make test-all` to run all tests
 - `make coverage` to run tests with coverage and generate a html report
 
+If not, you are able to manually run `poetry run pytest` or `pytest tests/`
```

Comment:
I think this section can be removed. The `Makefile` is very specific to the Starlite repo